demos/main/src/main/java/com/google/android/exoplayer2/demo/upstream/ClientUtil.java

```diff
@@ -0,0 +1,21 @@
+package com.google.android.exoplayer2.demo.upstream;
+
+import java.util.concurrent.TimeUnit;
+import okhttp3.OkHttpClient;
+
+public class ClientUtil {
```

Comment:
maybe rename to `OkHttpClientUtil` or `OkHttpClientFactory` ?

Reply:
done